data_convert/encryption.py

```diff
@@ -48,6 +48,15 @@ def decrypt(password: str, encrypted_msg: str) -> str:
         raise Exception("Bad Password")
     return msg
 
+def find_in_parent(file_name: str) -> str:
+    "locate a file in current directory or parents"
```

Comment:
it's better to do file management using where you'd expect it to be in the module directory (check out https://stackoverflow.com/questions/247770/how-to-retrieve-a-modules-path). But also, assuming you want to stay with the Sheets API and not download a CSV from a public link, maybe we should just use GitHub secrets instead of having the creds checked into the repo?

Reply:
Yeah, that's better but not worth worrying about right now.  We're going to remove the key soon.